packages/server/template.env

```diff
@@ -28,3 +28,6 @@ MAILGUN_API_HOST="api.eu.mailgun.net"
 # APP
 FOUNDER_ID="SET_VALUE" # user that will send initial welcoming in app message
 CONTACT_EMAIL="an email"
+
+# Local HTTPS
+LOCALHOST_CERTIFICATE_BASE_PATH=/Users/aleksmarkovic/Desktop/
```

Comment:
With this changes I'd like to add 1 more env variable to the web. 

`DEV_SERVER_URL` and use it `packages/web/src/setupProxy.js` instead of having hardcoded dev server url.